test/httpClient.test.ts

```diff
@@ -0,0 +1,6 @@
+import { integrationTest, readAppmap, runAppmapNode } from "./helpers";
+
+integrationTest("mapping http client requests", () => {
+  expect(runAppmapNode("yarn", "exec", "ts-node", "index.ts"));
```

Comment:
This will also record the server part, which is not of concern for this test. Please set up a server in this test file here instead (with plain node, not with express) and make sure the recorded script only acts as a client.